.jenkins/Jenkinsfile

```diff
@@ -44,8 +44,11 @@ def ACCContainerTest(String label, String version) {
     stage("Ubuntu ${version} Non-Simulation Container SGX1-FLC RelWithDebInfo") {
         node("${label}") {
             timeout(GLOBAL_TIMEOUT) {
+			
                 cleanWs()
                 checkout scm
+				
+                def dcap = load pwd() + "/.jenkins/src/Dcap.groovy"
```

Comment:
Same comment as above - can we put this at the top to avoid duplicate code?

Reply:
Unfortunately not. It should be put after checkout scm so it can find the groovy file.